### ISSUE TITLE
Update Travis integration?

### ISSUE THREAD

**Issue by uliska** [openlilylib/snippets#140]
I'm not sure what we have to do to make Travis work properly with the renamed repository.

_Do_ we even have to do anything?
The first failed test (https://travis-ci.org/openlilylib/openlilylib/builds/90966857) indicates that there's something fundamentally wrong. I see that this URL should contain openlilylib/snippets but I'm not sure if we have to update our repo files or (and) have to register something with Travis.

**Comment by Cecca** [comment]
Did you rename the `openlilylib/openlilylib` repository to `openlilylib/snippets`? If so, that may be the source of the problem.

**Comment by Cecca** [comment]
Oh sorry. You wrote exactly that in your message.. I just jumped directly to the error message on Travis. I think that the simplest thing to do then is to reset everything on the Travis side

**Comment by uliska** [comment]
Yes. I renamed it (in order to "move" the issue tracker and all the pull
request history to the new location) and created a mirror of it (only
the code) under the old name.

I'm sure that's the cause of the problem but I don't know out of my hat
where to deal with it. the `.travis.yml` seems completely agnostic of
the name.

**Comment by Cecca** [comment]
You could try to head to https://travis-ci.org/profile/openlilylib and flick the `openlilylib/snippets` switch on. I can't because I don't have admin access to this repository.

This should associate this newly named repository to Travis.
